Week1/work.py

```diff
@@ -37,8 +41,14 @@ def sqrt_of_numbers(num):
     '''
 
     ## Code Here
-    return None
-
+    
+    if num>0:
+        num=num
+    else:
+        num=-num
+    x=sqrt(num)
```

Comment:
the problem is there is no default function sqrt in python use ```math.sqrt()``` in place of that